src/memory/AmdFlash.hh

```diff
@@ -147,21 +154,25 @@ namespace AmdFlashChip
 	// AMD AM29F040
 	static constexpr Chip AM29F040 = {
 		.autoSelect{ManufacturerID::AMD, .device{0xA4}, .extraCode = 0x01},
+		.geometry{.regions{{{8, 64 * 1024}}}},
 	};
 
 	// AMD AM29F016
 	static constexpr Chip AM29F016 = {
 		.autoSelect{ManufacturerID::AMD, .device{0xAD}},
+		.geometry{.regions{{{32, 64 * 1024}}}},
 	};
 
 	// Numonyx M29W800DB
 	static constexpr Chip M29W800DB = {
 		.autoSelect{ManufacturerID::STM, .device{0x5B}},
+		.geometry{.regions{{{1, 16 * 1024}, {2, 8 * 1024}, {1, 32 * 1024}, {15, 64 * 1024}}}},
```

Comment:
I'm not sure. Does the code become more clear with a `kB()` helper function, or with a `_kB` user defined literal?
Here's a small experiment, I don't know if it's worth the trouble:
https://godbolt.org/z/o8vdGad6G

Reply:
Of the two I prefer the suffix. Or `_k` since the suffix can apply to more than just bytes. Or actually `_ki` since they’re not thousands.

But honestly not sure if it’s worth it, maybe too fancy / specific. Either way that’s more of an openMSX-wide change I think, should also consider in the context of what it does to the code base as a whole, whether it’s friendly or hostile to new contributors, etc.

Reply:
Replaced with hex values `0x2000`, `0x4000`, `0x8000` and `0x10000`. To me that’s easier to read without getting too fancy.